### DISCUSSION TITLE
Option to hide slug field for section

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#13799] [Ideas] [answered]
Sometimes we need to hardcode section entry url instead of constructing it using slug.
In that case, slug is pretty much useless. Why not add option to hide this field for specific section?

**Comment by brandonkelly** [comment]
Agree it makes sense to make it optional, probably as an entry type setting (similar to #12837).

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.0.0-alpha.4 was just released, with a new “Show the Slug field” entry type setting.
